components/UserTestimonies/ViewTestimony.tsx

```diff
@@ -122,16 +125,32 @@ export const FormattedTestimonyContent = ({
   const TESTIMONY_CHAR_LIMIT = 442
   const [showAllTestimony, setShowAllTestimony] = useState(false)
 
+  const _formatTestimony = (testimony: string, limit?: number) => {
+    const linkedTestimony = Autolinker.link(
+      limit ? testimony.slice(0, TESTIMONY_CHAR_LIMIT) : testimony,
+      {
+        truncate: 32
+      }
+    )
+
+    const paragraphedTestimony = linkedTestimony
+      .split(/[\r\n]+/)
```

Comment:
nit: use `/\s*[\r\n]+\s*/` to strip out whitespace around paragraphs (indentation)